snap/snapcraft.yaml

```diff
@@ -0,0 +1,489 @@
+    build-packages:
+      - python3-pip
+    prime:
+      - -*
+
+  mangohud:
+    after: [meson-deps]
+    source: https://github.com/flightlessmango/MangoHud.git
+    source-tag: 'v0.7.2'
+    source-depth: 1
+    plugin: meson
+    organize:
+      snap/heroic/current/usr: usr
+    build-environment:
+      - PKG_CONFIG_PATH: /usr/lib/i386-linux-gnu/pkgconfig:/usr/lib/x86_64-linux-gnu/pkgconfig:/usr/lib/pkgconfig
+    meson-parameters:
+      - --prefix=/usr
+      - --libdir=lib/i386-linux-gnu
+      - -Dappend_libdir_mangohud=false
+    build-packages:
+      - gcc
+      - gcc-multilib
+      - g++
+      - g++-multilib
+      - glslang-dev
+      - glslang-tools
+      - libdbus-1-dev
+      - libgl-dev
+      - libglvnd-dev
+      - libx11-dev
+      - libxnvctrl-dev
+      - mesa-common-dev
+      - python3-mako
+      - libwayland-dev
+      - libxkbcommon-dev
+    # Fixes the $LIB being escaped for some reason
+    override-build: |
+      craftctl default
+      sed -i 's/\/usr\/\\\$LIB/\$SNAP\/usr\/lib\/x86_64-linux-gnu/' $CRAFT_PART_INSTALL/usr/bin/mangohud
+
+  mangohud64:
+    after: [meson-deps, mangohud]
+    source: https://github.com/flightlessmango/MangoHud.git
+    source-tag: 'v0.7.2'
+    source-depth: 1
+    plugin: meson
+    organize:
+      snap/heroic/current/usr: usr
+    meson-parameters:
+      - --prefix=/usr
+      - --libdir=lib/x86_64-linux-gnu
+      - -Dappend_libdir_mangohud=false
+    stage:
+      - -usr/bin/mangohud
+      - -usr/share/vulkan/implicit_layer.d/MangoHud.json
+      - -usr/share/vulkan/implicit_layer.d/MangoHud.x86_64.json
+      - -usr/share/vulkan/implicit_layer.d/libMangoApp.json
+      - -usr/share/doc/mangohud/MangoHud.conf.example
+      - -usr/share/man/man1/mangohud.1
+
+  gamemode:
+    after: [meson-deps]
+    source: https://github.com/FeralInteractive/gamemode.git
+    source-tag: '1.8.2'
+    source-depth: 1
+    plugin: meson
+    organize:
+      snap/heroic/current/usr: usr
+    meson-parameters:
+      - --prefix=/usr
+    build-packages:
+      - libsystemd-dev
+      - pkg-config
+      - libdbus-1-dev
+    prime:
+      - usr/bin/gamemoderun
+      - usr/lib/*/libgamemode*.so.*
+
+  heroic:
+    source: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher.git
+    source-tag: 'v2.15.2'
+    source-depth: 1
+    plugin: dump
+    build-packages:
+      - dpkg-dev
+      - nodejs
+    prime:
+      - -chrome-*
+      - -chrome_crashpad_handler
+      - -usr/share/doc
+      - -usr/share/man
+      - -usr/share/bug
+      - -usr/share/gdb
+      - -usr/share/emacs*
+      - -usr/share/lintian
+      - -usr/share/drirc.d
+      - -usr/share/vulkan
+      - -usr/share/Xsession.d
+      - -usr/lib/*/dri
+      - -usr/lib/*/vdpau
+      - -usr/lib/*/libvkd3d*
+      - -usr/lib/*/libvulkan*
+      - -usr/lib/*/libVk*
+      - -usr/lib/*/libLLVM*
+    override-build: |
+      set -eux
+      patch -p1 < $CRAFT_PROJECT_DIR/dist.patch
```

Comment:
Where does this patch come from?

Reply:
Hmmm... probably the single line change that I have pushed in this PR too